pentasharp/ViewModel/Bus/EditBusViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using pentasharp.Models.Entities;
+using pentasharp.Models.Enums;
+using System.ComponentModel.DataAnnotations;
+
+namespace pentasharp.ViewModel.Bus
+{
```

Comment:
ctrl + a
ctrl + k + f to reformat the whole class.
Add xml summaries and also leave only one white space line in between properties